cohere/responses/tokenize.py

```diff
@@ -0,0 +1,53 @@
+from concurrent.futures import Future
+from typing import List, Optional
+
+from cohere.responses.base import CohereObject, _df_html
+
+from concurrent.futures import Future
+from typing import Optional
+
+import html
+from collections import UserList
+from dataclasses import asdict, dataclass
+from typing import Dict, List, Optional, Union
+
+from cohere.responses.meta_response import Meta
+
+class Tokens(CohereObject):
+
+    def __init__(self,
+                 tokens: Optional[List[int]] = None,
+                 token_strings: Optional[List[str]] = None,
+                 meta: Optional[Meta] = None) -> None:
+        assert tokens is not None
+        assert token_strings is not None
+        self.tokens = tokens
+        self.token_strings = token_strings
+        self.meta = meta
+
+    @property
+    def length(self):
+        return len(self)
+
+    def __len__(self) -> int:
+        return len(self.tokens)
+
+    def visualize(self, **kwargs):
+        import pandas as pd
```

Comment:
should we wrap this in a try/except and provide a custom error message?

Reply:
repr_html uses try/except (and returns a reasonable alternative), and this is only used there.